src/createServer.js

```diff
@@ -1,3 +1,66 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable no-console */
+
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+const createServer = () => http.createServer((req, res) => {
+  const url = new URL(req.url, `http://${req.headers.host}`);
+
+  const textToConvert = String(req.url).split('?')[0].slice(1);
```

Comment:
```suggestion
  const originalText = url.pathname.slice(1);
```
You can do it more efficient cause you have created new URL object. Also proper variable name can shorten some final code.